deployments/datatypes.go

```diff
@@ -162,3 +162,12 @@ func getDataTypeComplexType(dataType string) string {
 	}
 	return tType
 }
+
+func isPrimitiveDataType(dataType string) bool {
```

Comment:
Why not use https://github.com/ystia/yorc/blob/develop/tosca/types.go#L36  here?

Reply:
Didn't see this but seems weird map and list are considered like builtin type if entry shcema is not a primitive

Reply:
Fixed